.circleci/config.yml

```
@@ -6,7 +6,7 @@ defaults: &defaults
     GRUNTWORK_INSTALLER_VERSION: v0.0.21
     TERRATEST_LOG_PARSER_VERSION: v0.29.0
     MODULE_CI_VERSION: v0.28.3
-    TERRAFORM_VERSION: NONE
+    TERRAFORM_VERSION: 0.13.4
```

I think we could bump terraform in here and in terraform-aws-eks.

```suggestion
    TERRAFORM_VERSION: 0.13.5
```